Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/branch.py

```diff
@@ -113,14 +113,14 @@ def cc_radar(cls, args, repository, issue):
 
         radar_cc_default = repository.config().get('webkitscmpy.cc-radar', 'true') == 'true'
         if needs_radar and (args.cc_radar or (radar_cc_default and args.cc_radar is not False)):
-            rdar = None
-            if not getattr(args, 'defaults', None):
-                sys.stdout.write('Existing radar to CC (leave empty to create new radar)')
-                sys.stdout.flush()
-                input = Terminal.input(': ')
-                if re.match(r'\d+', input):
-                    input = '<rdar://problem/{}>'.format(input)
-                rdar = Tracker.from_string(input)
+            if not rdar:
```

Comment:
This will prompt you for a radar twice if there is no existing radar to cc. Going to fix this before landing.